src/app/modules/shared-components/components/page-navigator/page-navigator.component.ts

```diff
@@ -19,11 +33,45 @@ export class PageNavigatorComponent {
   @Output() watch = new EventEmitter<void>();
   @Output() stopWatch = new EventEmitter<void>();
 
+  @ViewChild('op') private op?: OverlayPanel;
+
+  overlayPanelMarginTop = true;
+
   isCurrentGroupWatched$ = this.userSessionService.session$.pipe(
     filter(isNotNullOrUndefined),
     map(data => this.groupId !== undefined && data.watchedGroup?.id === this.groupId)
   );
 
-  constructor(private userSessionService: UserSessionService) {
+  state$ = this.userSessionService.group$.pipe(
```

Comment:
So you fetch the list of root activities of the watched group when you arrive on a group page, even if you do not display the list of activities? Seems strange.

Reply:
No, on show tooltip it makes the call, so no requests until user clicked on eye icon